translations/monero-GUI-guide_es.md

```diff
@@ -0,0 +1,331 @@
+**(6) Exportar billetera gastable:** [**NO compartas tu código QR del monedero con nadie. Esto puede usarse como una semilla mnemotécnicas para recuperar su billetera.**] Esta opción crea un código QR que contiene todas sus claves.    
+**(7) Exportar solo billetera vista:** Esta opción crea un código QR que contiene solo las claves para ver las transacciones que esta billetera envía o recibe, pero no puede crear y generar transacciones.    
+
+# 7. Verificación de los binarios
+
+Debes asegurarte de que los ficheros que has descargado se corresponden con los ficheros oficiales. 
+Puedes utilizar [esta sencilla guía ilustrada](https://getmonero.org/resources/user-guides/verification-windows-beginner.html) para sistemas Windows.
+
+# 8. Problemas frecuentes y soluciones
+
++ **[[EN] He perdido (no estoy viendo) una transacción entrante en la interfaz gráfica (balance cero)](https://monero.stackexchange.com/questions/6640/i-am-missing-not-seeing-a-transaction-to-in-the-gui-zero-balance)**
++ **[[EN] Uso la interfaz gráfica y mi demonio ha dejado de arrancar](https://monero.stackexchange.com/questions/6825/i-am-using-the-gui-and-my-daemon-doesnt-start-anymore)**
++ **[[EN] La transacción está bloqueada como _pendiente_ en la interfaz gráfica](https://monero.stackexchange.com/questions/6649/transaction-stuck-as-pending-in-the-gui)**
++ **[[EN] La interfaz gráfica parece que tiene errores o se congela todo el rato](https://monero.stackexchange.com/questions/6651/my-gui-feels-buggy-freezes-all-the-time)**
++ **[[EN] Mi nombre contiene caracteres especiales (non-ASCII) (e.g. é, ø, â, Ö) y no puedo crear un monedero en la interfaz gráfica](https://monero.stackexchange.com/questions/6823/my-name-contains-a-special-non-ascii-character-e-g-%c3%a9-%c3%b8-%c3%a2-%c3%96-and-i-cant-c)**
++ **[[EN] La interfaz gráfica usa todo mi ancho de banda y no puedo navegar o usar otras aplicaciones que requieran conexión a internet](https://monero.stackexchange.com/questions/6653/the-gui-uses-all-my-bandwidth-and-i-cant-browse-anymore-or-use-another-applicat)**
++ **[[EN] ¿Cómo puedo mover la cadena de bloques (`data.mdb`) a un directorio diferente durante (o después) de la sincronización inicial sin perder los avances?](https://monero.stackexchange.com/questions/7225/how-do-i-move-the-blockchain-data-mdb-to-a-different-directory-during-or-afte)**
++ **[[EN] ¿Cómo puedo cambiar el idioma de las 25 palabras de la semilla mnemotécnica en la interfaz gráfica y en la terminal?](https://monero.stackexchange.com/questions/7373/how-do-i-change-the-language-of-the-25-word-mnemonic-seed-in-the-gui/)**
++ **[[EN] Mi cadena de bloques está bloqueada, ¿cómo la desatasco?](https://monero.stackexchange.com/questions/4462/my-blockchain-is-stuck-how-do-i-unstuck-it)**
```

Comment:
mmm.. "despego"? :stuck_out_tongue_closed_eyes: Not sure on this.

Reply:
Sorry, I don't understand what's the problem here! ;)

Reply:
No, you can leave it. Never pointed a problem. Just maybe a different word. "Desatasco" sounds a bit strange where I live but I know is a word that anyone can use.

Reply:
Hi @febrezo, lh1008 and me are two Spanish translators and we tend to review almost everything hehe, some of the reviews are more like suggestions, it is not a problem or something that must be changed yes or yes.

For this case I agree with lh1008, it does not sound bad and it's technically correct (*desatasco*) but at least in my country is not so much used, so what we mean by making some suggestions is to choose better terminology used in almost all the variants of the language (you know how diverse is the language).

I can suggest to use: * Mi cadena de bloques está bloqueada, ¿cómo la desbloqueo?*

To be honest I haven't read the original file, I'll look to it right now but I wish to leave the comment first, I can edit it later :)

Reply:
Ok, np! I didn't notice that «desatascar» was the problem. I'm updating it!